astroquery/alma/tests/test_alma.py

```diff
@@ -173,6 +173,15 @@ def test_gen_str_sql():
         "(proposal_id LIKE '2012.%' OR proposal_id LIKE '2013._3%')"
 
 
+def test_gen_array_sql():
+    common_select = "select * from ivoa.obscore WHERE "
```

Comment:
extreme nitpick: maybe add a comment that this tests the regression from #2094?

Reply:
Sure, thanks @bsipocz .  Will the `CHANGES.rst` entry fall under the `0.4.7` heading?

Reply:
yes, please